roles/ceph-common/tasks/checks/check_mandatory_vars.yml

```diff
@@ -22,6 +22,14 @@
   tags:
     - package-install
 
+- name: make sure ceph_stable_release is set
+  fail:
+    msg: "make sure ceph_stable_release is set to a release name, http://docs.ceph.com/docs/jewel/releases/"
+  when:
+    - ceph_stable_release == 'dummy'
```

Comment:
this looks naive enough that a typo in a release would go through.  Having an up to date set of current releases may not be something you want to maintain though.

What is the difference between this and just making sure it is not undefined?